### PR TITLE
Remove more JCenter references

### PR DESCRIPTION
**Pull request by cobexer**
### Reviewing cheatsheet

Before merging the PR, comments starting with 
- ❌ ❓**must** be fixed
- 🤔 💅 **should** be fixed
- 💭 **may** be fixed
- 🎉 celebrate happy things

### CODE REVIEW

**Comment by cobexer** [comment]
@bot-gradle test RFR

**Review by ljacomet — APPROVED** [review]
Makes sense from looking at the changes in the source repos.

**Comment by cobexer** [comment]
@bot-gradle test RFR without PTS

File: gradle/shared-with-buildSrc/mirrors.settings.gradle.kts [hunk 1]
```diff
@@ -24,6 +24,7 @@ import org.gradle.tooling.events.OperationCompletionListener
 
 class Helper(private val providers: ProviderFactory) {
     val originalUrls: Map<String, String> = mapOf(
+        "jcenter" to "https://jcenter.bintray.com/",
```
Comment (ov7a) [inline gradle/shared-with-buildSrc/mirrors.settings.gradle.kts:27]: 🤔 why do we want to preserve this?

**Comment by cobexer** [comment]
@bot-gradle test qf

File: gradle/shared-with-buildSrc/mirrors.settings.gradle.kts (hunk 1 above)
Comment (cobexer) [reply gradle/shared-with-buildSrc/mirrors.settings.gradle.kts]: Thanks for pointing this out, we don't need this for release/master

[the remark this answers is not in the document]
Comment (cobexer) [reply gradle/shared-with-buildSrc/mirrors.settings.gradle.kts]: As discussed OOB, we don't need this for release/master

**Review by ov7a — APPROVED** [review]

**Pull request merged**